.gems

```diff
@@ -1 +1,2 @@
 citrus -v 3.0.2
+minitest -v 5.7.0
```

Comment:
If you look on the unit::test documentation http://ruby-doc.org/stdlib-2.1.1/libdoc/test/unit/rdoc/Test/Unit.html it state `If you are writing new test code, please use MiniTest instead of Test::Unit.`